charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/elasticsearch-logging-role-binding.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: {{ include "rbacversion" . }}
+kind: RoleBinding
+metadata:
+  name: elasticsearch-logging
+  namespace: {{ .Release.Namespace }}
+  labels:
+    gardener.cloud/role: logging
```

Comment:
Please, add the old label key `garden.sapcloud.io`, too.
It is needed for the deletion of the logging resources.